src/main/webapp/script.js

```diff
@@ -28,7 +28,34 @@ function initMap() {
         streetViewControl: false,
         fullscreenControl: false,
     });
+    // Initialise places autocomplete in search bar
+    const searchbar = document.getElementById("location-search-bar");
+    const searchMarker = new google.maps.Marker({
+        map,
+        anchorPoint: new google.maps.Point(0,-29),
```

Comment:
what does the (0,-29) point/lead to?

Reply:
Not entirely sure, that's just what was on the example

Reply:
It's to do with positioning the icon image relative to the place

Reply:
I've added a comment explaining it